webui/react/src/hooks/useModal/Checkpoint/useModalCheckpointRegister.tsx

```diff
@@ -217,7 +221,7 @@ const useModalCheckpointRegister = ({ onClose }: Props = {}): ModalHooks => {
         type: ErrorType.Api,
       });
     }
-  }, [canceler.signal, modalState.visible]);
+  }, [canceler.signal, modalState.visible, canModifyModel]);
```

Comment:
i got `Warning: Maximum update depth exceeded. This can happen when a component calls setState inside useEffect, but useEffect either doesn't have a dependency array, or one of the dependencies changes on every render.`
we should avoid adding func in deps array.

https://user-images.githubusercontent.com/109113616/219453793-924be9b3-74ea-40dc-8a29-918c2ece691d.mov